lib/Tab.js

```diff
@@ -22,15 +22,20 @@ class Tab extends React.PureComponent {
     shouldRender: !this.props.lazy,
   };
 
+  getAdditionalStyles() {
+    const opacity = this.props.hideInvisible && this.props.selectedTabIndex !== this.props.tabIndex ? 0 : 1;
```

Comment:
Looks like you haven't used prettier. Make sure you have it installed and ran it just before submitting a PR, please.

Here is how should it looks like:

```js
  getAdditionalStyles() {
    const { hideInvisible, selectedTabIndex, tabIndex } = this.props;

    const opacity =
      hideInvisible && selectedTabIndex !== tabIndex ? 0 : 1;

    return {
      opacity,
    };
  }
```